### DISCUSSION TITLE
I cannot get my new users to see my images

### DISCUSSION THREAD

**Discussion by dlwhitehurst** [requarks/wiki#7000] [Help / Questions]
I've been running wikijs v2.5.301 for a couple of years on kubernetes with no storage status "you don't have any active storage target". I backup my Postgres container and I'm quite sure that the images (assets) are part of the database storage. I've recently created several users, including one for myself for testing, and I cannot get anyone to see images like I do as full admin. I've set group perms for all page and asset perms within my group space (folder). Everything works except the image is the little icon (not there) and /cat.jpg . I've also tested with full path e.g. https://wikijs.dlwhitehurst.com/cat.jpg and that works (again) for full admin but none of my users.

Can someone help here?

**Comment by dlwhitehurst** [comment]
Just adding that if I edit the image with full path, and navigate to the link, I get "forbidden" in the response. I tested with the group's content all checked and then set page properties as well to test. I also tried setting the security context in the deployment for root and that didn't work either.

**Reply by fnafox** [reply]
If you (admin) can see the content but other people cannot then it's a problem with permissions in wikijs, changing the security context for the deployment won't resolve the issue.

If you place the picture in the folder where the group has access to can they see it then? It's hard to tell what the issue is without seeing how your permissions are currently set up. Review the permissions and make sure your groups have read:pages and read:assets set on the desired paths and that a different rule isn't overriding it.

**Reply by dlwhitehurst** [reply]
1. I tried creating a folder using the UI, e.g. images, but that didn't seem to work. Now that you are mentioning permissions, I'm thinking maybe I need to set permissions on the "folder" path? E.g. I have group at /Business and landing at /Business/home. I then will try to add /Business/images and add the perm rules to that for all assets perms.

2. I had set and tried all perms for assets before on my site or group folder /Business. But ... you've sparked my mind to try the folder route and not just assume that the folder gets perms too.

**Reply by dlwhitehurst** [reply]
Update:
1. I removed the security context from the deployment. Yes, that wasn't the answer, that only allowed the container to run as root (insecure)
2. I realized that as admin my addition of the image was either /root/cat.jpg or /image/cat.jpg (relative path) and my users never had any access to anything prior to /Business. So I added <root>/landing page with a link to /Business/home so that meant I needed an "exactly" permission of read to /landing. I did that and then my users had access to <root>/cat.jpg specified in /Business/home.
